cdap-app-fabric/src/main/java/io/cdap/cdap/internal/profile/ProfileService.java

```diff
@@ -226,6 +251,8 @@ public void deleteAllProfiles(NamespaceId namespaceId)
       List<Profile> profiles = profileStore.getProfiles(namespaceId, false);
```

Comment:
With the way this is implemented, this could potentially leak information about what profiles exist back to the user. Can we change this to use our existing getProfiles call instead?

Reply:
done